pkg/provider/identity.go

```diff
@@ -332,16 +357,14 @@ func (fi *FullIdentity) DialOption() (grpc.DialOption, error) {
 		InsecureSkipVerify: true,
 		VerifyPeerCertificate: peertls.VerifyPeerFunc(
 			peertls.VerifyPeerCertChains,
-			func(_ [][]byte, parsedChains [][]*x509.Certificate) error {
-				return nil
-			},
-			// TODO(coyle): Check that the ID of the node we are dialing is the owner of the certificate.
+			VerifyPeerIdentity(id),
 		),
 	}
 
 	return grpc.WithTransportCredentials(credentials.NewTLS(tlsConfig)), nil
 }
 
+// TODO: (dylan) we should probably start to standardize this around dht.NodeID type
```

Comment:
[there's ticket for this now](https://storjlabs.atlassian.net/browse/V3-624), can we delete this comment?

Reply:
AH, I didn't know that ticket existed, I'll delete the comment

Reply:
:+1: